lib/pages/search/public_room/empty_search_public_room_widget_style.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class EmptySearchPublicRoomWidgetStyle {
+  static const double avatarSize = 56.0;
+
+  static const BorderRadiusGeometry avatarBorderRadius =
+      BorderRadius.all(Radius.circular(28.0));
+  static BoxBorder avatarBorder(BuildContext context) {
+    return Border.all(
+      color: Theme.of(context).brightness == Brightness.dark
+          ? const Color(0x222c2d2f)
+          : const Color(0x222c2d2f),
+      width: 1,
+    );
+  }
+
+  static Color avatarBackgroundColor(BuildContext context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? const Color(0x222c2d2f)
```

Comment:
maybe we could put this color as a const variable somewhere

Reply:
updated